docs/guided-tour/blueprints/external-blueprint/README.md

```diff
@@ -27,30 +27,15 @@ We have uploaded the blueprint
 registry, from where the Landscaper can access it. You can find the commands, which we have used to upload the blueprint
 in this script: [commands/blueprint.sh](./commands/blueprint.sh).
 
+## Component Versions
 
-## Components
+To consume the blueprint from such a referencable location, it has to be contained in a component version. A component 
+version is a concept introduced by the [Open Component Model](https://github.com/open-component-model/ocm). In short, technically, 
+a component version consists of a *component-descriptor* and a number of *blobs* (= arbitrary binary objects).  
 
-> **_NOTE:_** **To follow along the following section, be sure to set the `useOCM: true` feature switch in the 
-> values.yaml, as shown [here](https://github.com/gardener/landscaper/blob/master/docs/installation/install-landscaper-controller.md#configuration-through-valuesyaml).**
-
-To consume the blueprint from such a referencable location, it has to be contained in a component.
-
-A component is a concept introduced by the [Open Component Model](https://github.com/open-component-model/ocm). In
-short, technically, a component consists of a *component-descriptor* and a number of *blobs* 
-(= arbitrary binary objects).  
-The file system representation of our component used for this example is shown [here](./components/component). 
-The *component-descriptor* describes the overall component. As you can see in the *component-descriptor*, the component
-only contains a single resource - the *blueprint*. And we reference that blueprint at an external location, the *OCI
-registry*.  
-
-Alternatively, instead of referencing the blueprint at an external location, we could have embedded it in the
-component as a so-called local blob. Then, the *blob* directory in the file system representation would not have been 
-empty. It would have contained the blueprint (typically as a tar archive), as demonstrated 
-[here](./components/local-blob-component).
-
-The most convenient way to generate such components is through corresponding *component configuration files*. The
-*component configuration file* used to create the component for this example is shown 
-[here](./config-files/components.yaml): 
+The most convenient way to generate such component versions is through corresponding *component configuration files*. 
+The *component configuration file* used to create the component version for this example is shown
+[here](./config-files/components.yaml):
 
```

Comment:
I think we should cross reference https://ocm.software/docs/guides/getting-started-with-ocm

Reply:
It is on our follow-up list.